jdaviz/configs/imviz/plugins/footprints/preset_regions.py

```diff
@@ -92,17 +108,30 @@ def jwst_footprint(instrument, ra, dec, pa, v2_offset=0.0, v3_offset=0.0, apertu
     if not _has_pysiaf:
         raise ImportError('jwst_footprint requires pysiaf to be installed')
 
-    if instrument not in _instruments:  # pragma: no cover
-        raise ValueError(f"instrument must be one of {', '.join(_instruments.keys())}")
-
-    siaf_interface = pysiaf.Siaf(_instruments.get(instrument))
-
-    # Get center and PA offset from full aperture
-    full = siaf_interface.apertures[_full_apertures.get(instrument)]
-    corners = full.corners("tel", rederive=False)
-    v2 = np.mean(corners[0]) - v2_offset
-    v3 = np.mean(corners[1]) + v3_offset
-    pa_offset = full.V3IdlYAngle
+    if observatory not in _instruments:
+        raise ValueError(f"observatory must be one of {', '.join(_instruments.keys())}")
+
+    if instrument not in _instruments[observatory]:  # pragma: no cover
+        raise ValueError(f"instrument must be one of {', '.join(_instruments[observatory].keys())}")
+
+    siaf_interface = pysiaf.Siaf(_instruments.get(observatory).get(instrument))
+
+    # use different definitions of the center for JWST and Roman:
+    if observatory == "Roman":
+        if instrument.startswith("WFI"):
```

Comment:
It looks like both Roman instrument entries start with "WFI", when would the `else` condition be triggered?

Reply:
Great catch! This was leftover from a previous iteration. Fixed in [8a20a2b](https://github.com/spacetelescope/jdaviz/pull/3322/commits/8a20a2b7edb4483b969932582fea13f00d329013).